009_team_7_agent_design/009_team_7_agent_design.tex

```diff
@@ -1 +1,443 @@
+
+\section{Agent Design}
+\label{sec: Agent Design}
+There are 3 key elements that define the agent: its personality, behaviour and memory. The personality defines a given agent at its core, and for the purpose of this experiment is kept constant throughout a given simulation. An agents behaviour is influenced by its personality, memory (a record of its experiences) and the environment. The environment can include current floor, food available and interaction wit other agents. The personality and behaviour then combine to influence the various decisions that the agent makes over the course of the simulation. 
+This basic agent framework is illustrated in \Cref{agent_flow}.
+
+\begin{figure}[H]
+    \begin{center}
+        \includegraphics[width=6cm]
+        {009_team_7_agent_design/Images/agent_flow.png}
+    \end{center}
+    \caption{Agent framework.}
+    \label{agent_flow}
+\end{figure}
+
+\subsection{Personality Traits}
+\label{subsec: Agent Design}
+The use of personality traits to attribute human characteristics to our agents is a construct derived from Robert R. McCrae and Paul Costa's 5 factor theory of personality. McCrae and colleagues discovered that the big five traits are surprisingly universal which was confirmed by a study that looked at people from more than 50 different cultures. They determined that the five personality traits could be used to accurately model the human psychological framework.
+
+The five personality dimensions consist of: 
+\begin{itemize}
+    \item  \textbf{Openness}: The degree to which an individual is willing and eager to have new experiences. This also has implications for creativity and problem solving. An individual with low openness will behave more traditionally and will be less welcoming to change. For an agent in the tower this means a high openness score means that they are less impacted by floor changes and thus its behavioural characteristics are not varied as much. Conversely a low openness score means an agent is significantly impacted by change such as arriving at a lower floor which increases their greediness more than it would an agent with a higher openness score. Openness also influences responsiveness which dictates how much an agent is willing to accept or requests or treaties.
+    \item  \textbf{Conscientiousness}: The degree of conscientiousness of an individual determines their level of organization, discipline and thoughtfulness. Conscientious individuals are also more strategic and forward thinking. In the context of this experiment, a high conscientiousness score means an agent will demonstrate greater learning from its experiences, as it will identify underlying trends in these experiences in order to make strategically better decisions. A more planning-oriented agent of such will also give greater importance to collaborating with other agents.
```

Comment:
```suggestion
    \item  \textbf{Conscientiousness}: The degree of conscientiousness of an individual determines their level of organization, discipline and thoughtfulness. Conscientious individuals are also more strategic and forward thinking. In the context of this experiment, a high conscientiousness score means an agent will demonstrate greater learning from its experiences, as it will identify underlying trends in these experiences in order to make better decisions. A more planning-oriented agent of such will also give greater importance to collaborating with other agents.
```